packages/documentation-v7/src/stories/components/heading/heading.docs.mdx

```diff
@@ -0,0 +1,32 @@
+import { Meta, Canvas, Controls } from '@storybook/blocks';
+import { linkTo } from '@storybook/addon-links';
+import * as HeadingStories from './heading.stories';
+
+<Meta of={HeadingStories} />
+
+# Button
+
+<div className="lead">Use headings to structure content both visually and semantically.</div>
+
+All HTML headings are available, from `h1` the highest section level to `h6` the lowest.<br/>
+If you need to match the font styling of a heading but cannot use the associated HTML element, check out <a onClick={linkTo('utilities-heading--docs')}>heading utilities</a>.
+
+<Canvas of={HeadingStories.Default} />
+<div className="hide-col-default">
+  <Controls of={HeadingStories.Default} />
```

Comment:
I liked this approach with the `ArgTypes` so that the playground and the docs are trully separated. 
What do you think?

Reply:
I do like more the approach, where the user only has to visit one page (docs) and can see and do everything there.
Maybe this should be a team decision.

Reply:
We'll leave it as it is for now, because it could be possible, we'll switch to a docs-only mode in the future.